pkg/dockermanager/dockermanager.go

```diff
@@ -196,6 +196,8 @@ func (dm *Manager) Manage(
 	eventFilter.Add("type", "container")
 	eventFilter.Add("event", "start")
 	eventFilter.Add("event", "stop")
+	eventFilter.Add("event", "destroy")
+	eventFilter.Add("event", "die")
```

Comment:
Not sure if this was a problem for other runners, but on k8s we were getting a `stop` event only for the `pause` containers (which we ignore). All the `testplan` containers, were sending `die`.